app/scripts/modules/core/src/managed/ManagedResourceHistoryModal.tsx

```diff
@@ -4,14 +4,15 @@ import { DateTime } from 'luxon';
 import { UISref } from '@uirouter/react';
 
 import {
+  IModalComponentProps,
   ModalHeader,
   ModalBody,
   Table,
   TableRow,
   TableCell,
   standardGridTableLayout,
   useData,
-} from 'core/presentation';
+} from '../presentation';
```

Comment:
this is so that extending `IModalComponentProps` doesn't explode when TS can't transform the `core` alias in custom builds.